src/renderer/components/DebugMock.js

```diff
@@ -152,7 +152,8 @@ if (getEnv("MOCK")) {
   observerAwareEventLoop();
 }
 
-export const mockedEventEmitter = window.mock.events.emitter;
+// $FlowFixMe
```

Comment:
Why would we have to flow fixme here?

Reply:
because if I don't, then every time I'm using the mockedEventEmitter I would have to check it exists, even though we know it does.